frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -312,6 +319,52 @@ const HostSummary = ({
     );
   };
 
+  const renderOperatingSystemSummary = () => {
+    // No tooltip if minimum version is not set, including all Windows, Linux, ChromeOS operating systems
+    if (!osUpdatesData?.minimum_version) {
+      return (
+        <DataSet title="Operating system" value={summaryData.os_version} />
+      );
+    }
+
+    const tooltipMessage = (osVersion: string, minimumOsVersion: string) => {
+      const requirementMetTooltip = "Meets minimum version requirement.";
+      const requirementNotMetTooltip = (
+        <>
+          Does not meet minimum version requirement.
+          <br />
+          Deadline to update: {osUpdatesData.deadline}
+        </>
+      );
+
+      const removeOSPrefix = (version: string): string => {
```

Comment:
Maybe cleaner to just put this functionality directly into line 345 instead of defining a function?

Reply:
`osVersion.replace(...)`

Reply:
Or move this function definition to the top level or to helpers, to avoid the nested definitions

Reply:
Moved this to `utilities/helpers.ts` and added a test and comment since this is only for Apple OS